language-server/language-server-api/.js/src/main/scala/aqua/lsp/OutputTypes.scala

```diff
@@ -0,0 +1,86 @@
+package aqua.lsp
+
+import aqua.parser.lift.FileSpan
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.JSExportAll
+import scala.scalajs.js.{UndefOr, undefined}
+
+@JSExportAll
+case class CompilationResult(
+  errors: js.Array[ErrorInfo],
+  warnings: js.Array[WarningInfo] = js.Array(),
+  locations: js.Array[TokenLink] = js.Array(),
+  importLocations: js.Array[TokenImport] = js.Array(),
+  tokens: js.Array[TokenInfoJs] = js.Array()
+)
+
+@JSExportAll
+case class TokenInfoJs(location: TokenLocation, `type`: String)
```

Comment:
Can we call it `VarInfoJs` or maybe `ExprInfoJs`? Not every token has a type and it is confusing.